src/main/java/io/github/vhoyon/vramework/interfaces/LinkableCommand.java

```diff
@@ -1,26 +1,39 @@
 package io.github.vhoyon.vramework.interfaces;
 
-import io.github.vhoyon.vramework.objects.Request;
+import java.util.Arrays;
+
 import io.github.vhoyon.vramework.objects.ParametersHelp;
+import io.github.vhoyon.vramework.objects.Request;
 
 public interface LinkableCommand extends Command {
 	
-	Object getCalls();
+	String getCall();
+	
+	default String getActualCall(){
+		return this.getCall();
+	}
+	
+	default String[] getAliases(){
+		return new String[0];
+	}
 	
-	default String getDefaultCall(){
-		Object calls = getCalls();
+	default String[] getAllCalls(){
+		String[] allCalls = new String[getAliases().length + 1];
 		
-		if(calls == null)
-			return null;
+		allCalls[0] = getCall();
```

Comment:
The `getAllCalls` method should return `new String[0]` if `getCall()` returns null.